### PR TITLE
Use https for website source

### PR DESCRIPTION
**Pull request by ahocevar**
This pull request just changes a few links to https://, and fixes mixed content warnings on the 2.x website.

### CODE REVIEW

File: src/two/ccla.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 OpenLayers Corporate Contributor License Agreement ("Agreement") v1.0
-http://www.openlayers.org/ccla.txt
+https://openlayers.org/ccla.txt
```
Comment (fredj) [inline src/two/ccla.txt]: should be https://openlayers.org/two/ccla.txt

File: src/two/css/website.css [hunk 2]
```diff
@@ -55,7 +55,7 @@ background: none;
 border: none;
 color: black;
 text-decoration: none;
-background: #333 url(http://trac.openlayers.org/chrome/common/topbar_gradient2.png)  0 0 repeat-x;
+background: #333 url(https://trac.openlayers.org/chrome/common/topbar_gradient2.png)  0 0 repeat-x;
```
Comment (tschaub) [inline src/two/css/website.css]: I don't see a valid certificate for https://trac.openlayers.org/chrome/common/topbar_gradient2.png.

File: src/two/index.html [hunk 3]
```diff
@@ -47,18 +47,17 @@
   </div>
   <div id="navcontainer">
     <a class="navtabtraccurrent" href="/two/">Home</a>
-    <a class="navtab" href="http://trac.openlayers.org">Support &amp; Development</a>
+    <a class="navtab" href="https://trac.openlayers.org">Support &amp; Development</a>
```
Comment (tschaub) [inline src/two/index.html]: We should probably update this URL.  Not a valid certificate for https://trac.openlayers.org (this comes from osgeo.org).

File: src/two/index.html [hunk 4]
```diff
@@ -72,22 +71,22 @@
...
-                    <a href="http://www.opensourcepress.de/de/produkte/OpenLayers/159/978-3-937514-92-5">OpenLayers (in German)</a><div class="author">Till Adams &amp; Marc Jansen</div>Available from <a href="http://www.opensourcepress.de/de/produkte/OpenLayers/159/978-3-937514-92-5">Open Source Press</a>.
+                    <a href="https://www.opensourcepress.de/de/produkte/OpenLayers/159/978-3-937514-92-5"><img src="images/openlayers_de_1ed.png"></a>
+                    <a href="https://www.opensourcepress.de/de/produkte/OpenLayers/159/978-3-937514-92-5">OpenLayers (in German)</a><div class="author">Till Adams &amp; Marc Jansen</div>Available from <a href="https://www.opensourcepress.de/de/produkte/OpenLayers/159/978-3-937514-92-5">Open Source Press</a>.
                 </li>
             </ul>
         </div>
         <div class="cleared spaced">
-            <a href="http://www.osgeo.org/" class="centered"><img src="images/osgeo_sm.png"></a>
-            <p>OpenLayers is a project of the <a href="http://www.osgeo.org/">Open Source Geospatial Foundation</a>.</p>
+            <a href="https://www.osgeo.org/" class="centered"><img src="images/osgeo_sm.png"></a>
```
Comment (tschaub) [inline src/two/index.html]: Not a problem, but funny that https://www.osgeo.org/ returns a 302 with http://www.osgeo.org/home as the location (not https).

**Pull request merged**